### ISSUE TITLE
Possible to use in the browser?

### ISSUE THREAD

**Issue by rmilejcz** [react-dnd/react-dnd#916]
I'm building a single react component as an addition to a clients website, and I would love to use this library. However, the component I'm building is self-contained in the browser (has to be this way, unfortunately) so I don't have access to node features like export/import. 

Would it be possible to implement this library in a browser only environment? I included ReactDnD and ReactDnDHTML5Backend via unpkg, and recreated the steps in the tutorial to the best of my ability.
When I load my component I see these console errors:

`The above error occurred in the <Table> component: in Table Consider adding an error boundary to your tree to customize error handling behavior. You can learn more about error boundaries at https://fb.me/react-error-boundaries.` 

and 

`Error: Expected the backend to be a function or an ES6 module exporting a default function. Read more: http://react-dnd.github.io/react-dnd/docs-drag-drop-context.html`

Here is the basic gist of my code:

`    function collect(connect, monitor){
        return {
          connectDragSource: connect.dragSource(),
          isDragging: monitor.isDragging()
        }
      }

      const rowSource = {
        beginDrag(props){
          return {
            id: props.data.ID
          }
        }
      }

      const ItemTypes = {
        ROW: 'row'
      }

      class Row extends React.Component{
        render(){
          const { isDragging, connectDragSource, data } = this.props;
          return ReactDnD.connectDragSource(
            <tr style={{opacity: isDragging ? 0.5 : 1}}>
              <td>{data.NAME}</td>
              <td>{data.ACTIVITY}</td>
              <td>{data.LOCATION}</td>
            </tr>
          )
        }
      }

      class Table extends React.Component{
        render(){
          return ReactDnD.DragDropContext(ReactDnDHTML5Backend.HTML5Backend)(TechList)
        }
      }

      ReactDOM.render(
        <Table />,
        document.getElementById('root')
      );`

And my unpkg script tags

`<script src="https://unpkg.com/react-dnd@2.5.4/dist/ReactDnD.min.js"></script>`

`<script src="https://unpkg.com/react-dnd-html5-backend@2.5.4/dist/ReactDnDHTML5Backend.min.js"></script>`

Obviously this is not how the library was intended to be used but if it is possible I'd love to know what I'm doing wrong with the above code!

**Comment by froatsnook** [comment]
Hey @robbiemilejczak, using it in the browser is totally normal!  You can convert javascript using import/export to javascript all browsers understand using something like [babel](https://babeljs.io/).  I normally use webpack with babel-loader so that I can create a single javascript file which includes all of my code, as well as all of the dependencies (like react, react-dnd, etc.).  That way you don't have to include them as `script`s separately.  But there are lots of ways to solve this problem.

I've never used unpkg or tried to use react-dnd without using webpack, so I'm not positive I can help you get all set up, but I do see two things that look wrong in your code.

1. Try logging the value of `ReactDnDHTML5Backend.HTML5Backend` and `ReactDnDHTML5Backend`.  Since my code looks like:

```javascript
import HTML5Backend from "react-dnd-html5-backend";
```

I'm guessing that you should be using `ReactDnDHTML5Backend` or maybe `ReactDnDHTML5Backend.default`.  This would explain your error message.

2. You should have:

```javascript
class TechList extends Component {
   ...
}

TechList = ReactDnD.DragDropContext(ReactDnDHTML5Backend)(TechList);

class Table extends Component {
  render() {
    return <TechList />;
  }
}
```

in other words, you are correctly wrapping `TechList`, but you want to return it wrapped in `< />` so that `React.createElement` will be called.

**Comment by magicops** [comment]
Hello @froatsnook,
I used react-dnd version 4.0.5 in one of my projects two years ago and it used to have a single JS file in the Dist folder to be able to use in the browser directly. But in the recent releases, it has been removed and there are two CJS and ESM folders including multiple files using import/export or "require" that prevent me to use it in the browser.
Is there any possibility to have the bundled version back?
Or if not, would you please axaplaine how I can create that?
